puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/deploy_yum.yaml

```diff
@@ -0,0 +1,33 @@
+- job:
+    name: deploy_yum
+    project-type: workflow
+    logrotate:
+      daysToKeep: 2
+      numToKeep: -1
+    triggers:
+      - timed: "" # ???
+    paramters:
+      - string:
+          name: os
+          default: el6
```

Comment:
Let's update this OS to el7

Reply:
description could be updated as well then